lib/vrl/stdlib/src/parse_xml.rs

```diff
@@ -338,7 +339,10 @@ fn process_node<'a>(node: Node, config: &ParseXmlConfig<'a>) -> Value {
         NodeType::Root => Value::Object(recurse(node)),
 
         NodeType::Element => {
-            match (config.always_use_text_key, node.attributes().is_empty()) {
+            match (
+                config.always_use_text_key,
+                node.attributes().len().is_zero(),
+            ) {
```

Comment:
`Node::attributes` is now an iterator rather than a slice, and `is_empty()` appears to be unstable for iterators.

Reply:
Does `node.attributes().len() == 0` cause an error? The `.is_zero()` method is a little surprising here.

Reply:
No, either worked (IIRC) - I just stylistically preferred the method, happy to change it though

Reply:
For other readers, since I was confused, the attributes also implements `FixedSizeIterator` to provide the `len` method.